### DISCUSSION TITLE
How to get an aggregate shap values for a multidimensional feature?

### DISCUSSION THREAD

**Discussion by LALBJ** [shap/shap#2963] [Q&A]
I have a feature that is 64-dimensional, which means I can obtain a SHAP value for each dimension. However, I am unsure of how to calculate an aggregate SHAP value for this feature. Do you have any suggestions?

**Comment by jainraj** [comment]
You could add the SHAP values across the dimensions which make up a logical feature e.g., add up embedding layer SHAP values to get SHAP for the word (very common trick - [TDS blog by Conor Sullivan](https://towardsdatascience.com/shap-for-categorical-features-7c63e6a554ea), [Fiddler.AI blog](https://www.fiddler.ai/blog/should-you-explain-your-predictions-with-shap-or-ig))
